apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../../assets/css/color.scss";
+.total-amount {
+    font-size: 18px;
+}
+
+.customer-due-text {
+    color: $niagara;
+}
+
+.vendor-due-text {
+    color: $brink-pink;
+}
+
+span.month-dropdown {
+    font-size: 12px;
+    color: $dark-gray;
+    vertical-align: top;
+}
+
+.dashboard-filter .btn-group {
+    display: block;
+    line-height: 1;
+}
+
+.dashboard-filter .btn-group span {
+    vertical-align: middle;
+    color: $dark-gray;
+}
+
+.underline {
```

Comment:
underline class already using the remove it

Reply:
I am not write this new css, just separating css from TS file